src/network/featureflags.h

```diff
@@ -38,6 +38,14 @@ struct ServerFeatures {
 		// Server can parse bundle packets: `PACKET_BUNDLE` = 100 (0x64).
 		PROTOCOL_BUNDLE_SUPPORT,
 
+		// Server can parse bundle packets with compact headers and packed IMU rotation/acceleration frames:
+		// - `PACKET_BUNDLE_COMPACT` = 101 (0x65),
+		// - `PACKET_ROTATION_AND_ACCELERATION` = 23 (0x17).
+		PROTOCOL_BUNDLE_COMPACT_SUPPORT,
```

Comment:
This bit is already taken by https://github.com/SlimeVR/SlimeVR-Server/blob/819d2fc3b8c48936b622fd604eaf7aa4876c2bb4/server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FeatureFlags.kt#L54.